build.sbt

```diff
@@ -29,6 +29,13 @@ val commonJSSettings = Seq(
   }
 ) ++ browserTestSettings
 
+val commonNativeSettings = commonSmlBuildSettings ++ ossPublishSettings ++ Seq(
+  organization := "com.softwaremill.sttp",
+  scalaVersion := "2.11.12",
```

Comment:
So scala native is based on 2.11? What about 2.12?

Reply:
There is no 2.12 support. 
2.12 support is tracked here: https://github.com/scala-native/scala-native/issues/233